pom.xml

```diff
@@ -35,7 +35,7 @@
     <url>https://github.com/eclipse/microprofile</url>
     <inceptionYear>2017</inceptionYear>
     <organization>
-        <name>Eclipse MicroProfile</name>
+        <name>MicroProfile</name>
         <url>http://www.eclipse.org/microprofile</url>
```

Comment:
Should we put a `<post>` link for the microprofile-wg email list?

Reply:
We added